### ISSUE TITLE
Invalid byte sequence in UTF-8 in .typl8-pull-quote

### ISSUE THREAD

**Issue by nofont** [typeplate/starter-kit#39]
I've experienced some issues with the pull-quote encoding in environments and processors that require strict UTF-8 encoding.

To prevent this a suggestions is the encode the “ and ” to &ldquo; and &rdquo;

So this section: 

.typl8-pull-quote:before {
  content: '“';
  top: 0;
  left: 0;
}
.typl8-pull-quote:after {
  content: '”';
  bottom: 0;
  right: 0;
}

Would be this:

.typl8-pull-quote:before {
  content: '&ldquo;';
  top: 0;
  left: 0;
}
.typl8-pull-quote:after {
  content: '&rdquo;';
  bottom: 0;
  right: 0;
}

**Comment by grayghostvisuals** [comment]
Wow. Just out of curiosity do you have the utf-8 meta tag in the head of your doc? What about in the stylesheet?

**Comment by nofont** [comment]
Yes, head is equipped with <code>meta http-equiv="Content-Type" content="text/html; charset=UTF-8"</code>. The stylesheet is Typeplate 2 included with the usual <code>link rel="stylesheet" ...</code>. 

But the issue I reported is a bit of an edge-case. When I use Typeplate 2 the usual way (as above) everything works just fine in the browser. 

It's when I use an app like (Hammer)[http://hammerformac.com/] to compile html/css/sass and such into a static site that the issues appears. Hammer compiles each file by itself and it complains about the UTF-8 encoding.

**Comment by grayghostvisuals** [comment]
This sounds more like a fault on Hammer's part then our end…no?
